controllers/azurecluster_reconciler.go

```diff
@@ -36,8 +36,8 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
 )
 
-// azureClusterReconciler is the reconciler called by the AzureCluster controller
-type azureClusterReconciler struct {
+// azureClusterService is the reconciler called by the AzureCluster controller
+type azureClusterService struct {
```

Comment:
thanks for fixing that, I've been meaning to do it for a while

Reply:
I think this may overlap with https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1075

Reply:
Yeah, one of us has to rebase. Happy to do it after #1075 gets merged.

/cc @surajrana93